From: Hans van Kranenburg <hans@knorrie.org>
Date: Thu, 26 Nov 2020 16:06:03 +0100
Subject: t/h/L/vif-common.sh: fix handle_iptable return value

A return statement without explicit value will return the value of the
last command executed before this line with return was encountered.

This is not what we want. return 0.

Closes: #955994
Fixes: 2e0814f971dd ("vif-common: disable handle_iptable")
Reported-by: Samuel Thibault <sthibault@debian.org>
Signed-off-by: Hans van Kranenburg <hans@knorrie.org>
---
 tools/hotplug/Linux/vif-common.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/hotplug/Linux/vif-common.sh b/tools/hotplug/Linux/vif-common.sh
index 14cd44c..99ff1ba 100644
--- a/tools/hotplug/Linux/vif-common.sh
+++ b/tools/hotplug/Linux/vif-common.sh
@@ -183,7 +183,7 @@ frob_iptable()
 handle_iptable()
 {
   # This function is disabled in Debian packaging. See Debian bug #894013.
-  return
+  return 0
 
   # Check for a working iptables installation.  Checking for the iptables
   # binary is not sufficient, because the user may not have the appropriate
